docs/api/fetchBaseQuery.md

```diff
@@ -130,8 +130,7 @@ export const customApi = createApi({
 ```
 
 ### Handling non-standard Response status codes
-
-By default, we will `reject` any `Response` that has a status code that is `< 200` or `> 299` and set it to an error. This is the same behavior you've most likely experienced with `axios` and other popular libraries. In the event that you have a non-standard API you're dealing with, you can use the `validateStatus` option to customize this behavior.
+By default, we will `reject` any `Response` that does not have a status code of `2xx` and set it to an error This is the same behavior you've most likely experienced with `axios` and other popular libraries. In the event that you have a non-standard API you're dealing with, you can use the `validateStatus` option to customize this behavior.
```

Comment:
:hand: I'd prefer to change all uses of "we" to "RTK Query" or "the library" - ie, describe it in terms of the _code_ doing things, not "us" doing things.

Reply:
I think I covered the majority of it. I'll do another pass in the future and have made a mental note to skip out on the `we` everywhere :D